bridges/DockerHubBridge.php

```diff
@@ -17,12 +18,42 @@ class DockerHubBridge extends BridgeAbstract {
 				'required' => true,
 				'exampleValue' => 'rss-bridge',
 			)
-		)
+		),
+		'Official Image' => array(
+			'repo' => array(
+				'name' => 'Repository',
+				'type' => 'text',
+				'required' => true,
+				'exampleValue' => 'postgres',
+			)
+		),
 	);
 
 	const CACHE_TIMEOUT = 3600; // 1 hour
 
 	private $apiURL = 'https://hub.docker.com/v2/repositories/';
+	private $urlRegex = '/hub\.docker\.com\/r\/([\w]+)\/([\w-]+)/';
+
+	public function detectParameters($url) {
+		$params = array();
+
+		// user submitted image
+		if(preg_match($this->imageUrlRegex, $url, $matches)) {
```

Comment:
imageUrlRegex is not defined

Reply:
Oops, not sure how I missed this.  🤦‍♀️ 

see: https://github.com/RSS-Bridge/rss-bridge/pull/1996/commits/5a8d30a6e1c9f8fbbfe1b9368bf4583e0f949cf8